src/main/java/games/GameType.java

```diff
@@ -117,11 +186,12 @@ public GameType stringToGameType(String game) {
     /**
      * Creates an instance of the given game type, with a specific number of players and game seed.
      * Add here all games implemented.
+     *
      * @param nPlayers - number of players taking part in the game, used for initialisation.
-     * @param seed - seed for this game.
+     * @param seed     - seed for this game.
      * @return - instance of Game object; null if game not implemented.
      */
-    public Game createGameInstance(int nPlayers, long seed) {
+    public Game createGameInstance(int nPlayers, long seed, AbstractParameters params) {
```

Comment:
Since we're passing params here, should probably use it within the method - potentially checking if null to create new default params (as currently), otherwise use the ones passed?

Reply:
Hmm....I definitely recall doing exactly that. I seem to have lost some code snippets somewhere along the way. I shall resurrect them.

Good spot!